kpops/components/streams_bootstrap/streams/streams_app.py

```diff
@@ -28,8 +28,14 @@ class StreamsAppCleaner(KafkaAppCleaner):
     def helm_chart(self) -> str:
         return f"{self.repo_config.repository_name}/{AppType.CLEANUP_STREAMS_APP.value}"
 
+    @override
+    async def reset(self, dry_run: bool) -> None:
+        self.app.streams.delete_output = False
+        await super().clean(dry_run)
```

Comment:
shouldn't it be 
```suggestion
        await super().reset(dry_run)
```

Reply:
This is a bit confusing, please have a look at [this code on main](https://github.com/bakdata/kpops/blob/main/kpops/components/streams_bootstrap/streams/streams_app.py#L132). Since the cleanup job is done by streams bootstrap we just need to set the delete output to flase and deploy the clean job.

Reply:
I guess you're changing this anyways in #498 with the `reset` separate method so we can discuss it afterwards there 😄

Reply:
yes, but I wanted to keep everything as is for this PR 😄